### ISSUE TITLE
Use unique struct comment tag name

### ISSUE THREAD

**Issue by ftKnox** [arangodb/go-driver#44]
It would be beneficial and make it more inline with many other go drivers for other databases to use a unique name for the json marshal/unmarshalling. One specific use case is have a particular field used as a query field but I don't want that field saved to the database.

```go
type Container {
  ParentContainerID string `
}
```

**Comment by ewoutp** [comment]
Let me see if I understand you correctly.
You want an additional tag that overwrites the `json` tag when both exist. Right?

I assume the semantics of that tag should be identical to the `json` tag.

If so. This is something that we have been discussing. I'll revive that discussion.

**Comment by ftKnox** [comment]
Yeah it's how the gorethink and mgo drivers work for rethinkdb and mongodb.
